prowler/lib/powershell/powershell.py

```diff
@@ -191,8 +191,15 @@ def error_reader_thread():
         error_thread.daemon = True
         error_thread.start()
 
-        result = result_queue.get(timeout=timeout) or default
-        error_result = error_queue.get(timeout=1)
+        try:
+            result = result_queue.get(timeout=timeout) or default
+        except queue.Empty:
+            result = default
+
+        try:
+            error_result = error_queue.get(timeout=1)
+        except queue.Empty:
+            error_result = ""
```

Comment:
Maybe we can change the code like this?:

```suggestion
        try:
            result = result_queue.get(timeout=timeout) or default
            error_result = error_queue.get(timeout=1)
        except queue.Empty:
            result = default
```

Reply:
Of course. Thanks for the suggestion.